src/sentry/models/__init__.py

```diff
@@ -1,3 +1,4 @@
+from ..users.models.email import *  # NOQA
```

Comment:
```suggestion
from sentry.users.models.email import *  # NOQA
```

Lets stick to absolute imports. They are more portable and easier to understand.

Reply:
guh thank you for catching this, I need to make a checklist so I don't keep repeating this 😭